tests/typescript/class/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -151,6 +178,34 @@ class ExtendsAbstractClassAndImplementsInterfaces5
     Interface7,
     Interface8 {}
 
+class ImplementsInterfaceAndExtendsAbstractClass1<Foo>
+  extends FOOOOOOOOOOOOOOOOO
+  implements FOOOOOOOOOOOOOOOOO, BARRRRRRRRRR {}
+
+class Foo<FOOOOOOOOOOOOOOOOOOOOOOOOOOO, FOOOOOOOOOOOOOOOOOOOOOOOOOOO>
+  implements Foo {}
+
+class ImplementsInterfaceAndExtendsAbstractClass2<
+    TypeArgumentNumberOne,
+    TypeArgumentNumberTwo,
+    TypeArgumentNumberThree
+  >
+  extends FOOOOOOOOOOOOOOOOOO
+  implements BaseInterface {}
+
+class ImplementsInterfaceClass1<
+  TypeArgumentNumberOne,
+  TypeArgumentNumberTwo,
+  TypeArgumentNumberThree
+> implements BaseInterface {}
+
+class ImplementsInterfaceClassWithComments1<
+    TypeArgumentNumberOne,
+    TypeArgumentNumberTwo,
+    TypeArgumentNumberThree
+  > // comments
+  implements BaseInterface {}
```

Comment:
Why not:
```ts
class ImplementsInterfaceClassWithComments1<
  TypeArgumentNumberOne,
  TypeArgumentNumberTwo,
  TypeArgumentNumberThree
> 
  // comments
  implements BaseInterface {}
```

or
```ts
class ImplementsInterfaceClassWithComments1<
  TypeArgumentNumberOne,
  TypeArgumentNumberTwo,
  TypeArgumentNumberThree
> // comments
  implements BaseInterface {}
```
```

Reply:
Because to follow test for similar behavior: https://github.com/prettier/prettier/blob/master/tests/typescript/interface/__snapshots__/jsfmt.spec.js.snap#L170-L175

Reply:
But I don't know it was intended.